src/VerifyCustomFactorController.js

```diff
@@ -31,10 +31,20 @@ export default FormController.extend({
       // set the initial value for remember device (Cannot do this while defining the
       // local property because this.settings would not be initialized there yet).
       this.set('rememberDevice', rememberDevice);
+
+      if (this.settings.get('features.skipIdpFactorVerificationBtn')) {
+        this.set('provider', 'CUSTOM');
+        this.set('factorType', 'claims_provider');
+        this.save();
+      }
     },
 
     save: function() {
       const rememberDevice = !!this.get('rememberDevice');
+      const self = this;
+      if (this.settings.get('features.skipIdpFactorVerificationBtn')) {
+        this.trigger('request');
```

Comment:
why do we need to trigger a request? If I am following this right, when we automatically call save in `initialize` the request will be made as if we had clicked the button (if that is not true, then maybe we need to move that code to `postRender`?